cmd/opampsupervisor/supervisor/supervisor.go

```diff
@@ -260,14 +271,34 @@ func initTelemetrySettings(logger *zap.Logger, cfg config.Telemetry) (*component
 		),
 	)
 	if err != nil {
-		return nil, err
+		return telemetrySettings{}, err
+	}
+
+	var lp log.LoggerProvider
+	if len(cfg.Logs.Processors) > 0 {
```

Comment:
can this re-use the code in core for instantiating the logger provider?

Reply:
This PR copies [this code](https://github.com/open-telemetry/opentelemetry-collector/blob/eabab751a0d10d0c0614bb6d03db854f8bf67cf6/service/telemetry/logger.go#L42-L57) from Core. Is there a public method I could use instead?

Reply:
it's probably ok as is for now... would be nice to share it, but i also dont want to make an api public at this point